support/filters.py

```diff
@@ -0,0 +1,20 @@
+from django_filters import rest_framework
+
+from support import models
+
+
+class LayersGroupFilter (rest_framework.FilterSet):
+    """LayersGroupFilter data.
+
+    Filters:
+        category (int): catagoreis groups list
```

Comment:
categoreis?

Reply:
Correction completed and APP-support/views.py fixed imports.

Reply:
still categoreis...

Reply:
ok